betelgeuse/parser.py

```diff
@@ -187,3 +187,31 @@ def parse_docstring(docstring=None):
             field_value = output
             fields_dict[field_name] = field_value
     return fields_dict
+
+
+def parse_markers(all_markers=None, config=None):
+    """Parse the markers."""
+    resolved_markers = []
+    ignore_list = getattr(config, 'MARKERS_IGNORE_LIST', None)
+
+    def _process_marker(marker):
+        # Fetching exact marker name
+        marker_name = marker.split('mark.')[-1] if 'mark' in marker else marker
+
+        # ignoring the marker if in ignore list
+        if ignore_list and not any(
+                ignore_word in marker_name for ignore_word in ignore_list):
+            resolved_markers.append(marker_name)
+
+    for sec_marker in all_markers:
+        # If the marker is none
+        if not sec_marker:
+            continue
+        elif isinstance(sec_marker, list):
+            for marker in sec_marker:
+                _process_marker(marker)
+        else:
+            _process_marker(sec_marker)
+
+    resolved_markers = ', '.join(resolved_markers)
+    return resolved_markers
```

Comment:
```suggestion
    return ', '.join(resolved_markers)
```

Reply:
Just to simply the return, I made that ! But I can do the change !